contracts/protocol/PositionHouse.sol

```diff
@@ -37,7 +37,7 @@ contract PositionHouse is
         address trader,
         int256 quantity,
         uint256 leverage,
-        uint256 priceMarket,
+        uint256 entryPrice,
```

Comment:
in the formula, this function calculated by totalQuoteAmount / quantity so it equal to average entry price of this market order